Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM jeanblanchard/alpine-glibc
```

Comment:
why not use golang 1.5 docker image directly?
See https://github.com/docker-library/docs/tree/master/golang